src/asymmetric_crypto_ctx.cpp

```diff
@@ -127,15 +117,15 @@ namespace mococrw {
         /*
          * Constructor that checks the type of the given key
          */
-        RSAImpl(const Key& key, std::shared_ptr<PaddingBase> padding) : key(key), padding(padding) {
+        RSAImpl(const Key& key, std::shared_ptr<PaddingBase> padding) : _key(key), _padding(padding) {
```

Comment:
Please remove the issue references from the commit messages – they don't make sense, since this is open source code and nobody has access to our internal ticket system.